girder/api/v1/folder.py

```diff
@@ -50,6 +50,9 @@ def __init__(self):
     @filtermodel(model='folder')
     @autoDescribeRoute(
         Description('Search for folders by certain properties.')
+        .notes('To have a valid request you need to pass either a folderId or a text field.  '
+               'If you omit one of these parameters the request will fail and respond : '
+               '"Invalid search mode."')
```

Comment:
Let's make the sentence:

> You must pass either a "folderId" or "text" field to specify how you are searching for folders.

Reply:
Only this sentence or can I keep the 2 last sentence too ?